sites/partners/src/applications/PaperApplicationForm/sections/FormDemographics.tsx

```diff
@@ -30,8 +29,50 @@ const FormDemographics = () => {
     }))
   }, [register])
 
+  // Does a key exist in a root field or a sub array
+  const isKeyIncluded = (raceKey: string) => {
+    let keyExists = false
+    formValues?.race?.forEach((value) => {
+      if (value.includes(raceKey)) {
+        keyExists = true
+      }
+    })
+    return keyExists
+  }
+
+  // Get the value of a field that is storing a custom value, i.e. "otherAsian: Custom Race Input"
+  const getCustomValue = (subKey: string) => {
+    const customValues = formValues?.race?.filter((value) => value.split(":")[0] === subKey)
+    return customValues?.length ? customValues[0].split(":")[1]?.substring(1) : ""
+  }
+
+  const raceOptions = useMemo(() => {
+    return Object.keys(raceKeys).map((rootKey) => ({
+      id: rootKey,
+      label: t(`application.review.demographics.raceOptions.${rootKey}`),
+      value: rootKey,
+      register,
+      additionalText: rootKey.indexOf("other") >= 0,
+      defaultChecked: isKeyIncluded(rootKey),
+      subFields: raceKeys[rootKey].map((subKey) => ({
+        id: subKey,
+        label: t(`application.review.demographics.raceOptions.${subKey}`),
+        value: subKey,
+        defaultChecked: isKeyIncluded(subKey),
+        additionalText: subKey.indexOf("other") >= 0,
+        defaultText: getCustomValue(subKey),
+      })),
+    }))
+  }, [register])
```

Comment:
I don't think this properly handles the `Other / Multiracial` option. raceKeys doesn't have the dynamic value of `otherMultiracial`, so it never hits subFields or the `getCustomValue` call for defaultText to handle a value like `otherMultiracial: test other race`.

Reply:
Are you seeing that while testing in the UI? It seems to be functional

Reply:
Touched base offline, fixed!